common/thrift_client_pool.h

```diff
@@ -446,11 +439,11 @@ class ThriftClientPool {
 
  private:
   std::vector<EventLoop> event_loops_;
+  std::shared_ptr<folly::SSLContext> ssl_ctx_;
```

Comment:
As mentioned in the previous comment, we need to make it std::shared_ptr<folly::SSLContext>&, otherwise it's can't be modified externally.